queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -142,6 +143,20 @@ def process_email(email_msg: dict, flask_app: Flask):  # pylint: disable=too-man
         elif etype and etype == 'bc.registry.bnmove':
             email = bn_notification.process_bn_move(email_msg, token)
             send_email(email, token)
+        elif etype and etype == 'bc.registry.dissolution':
+            email = involuntary_dissolution_stage_1_notification.process(email_msg, token)
+            # Confirm the data.furnishingName
+            furnishing_name = email_msg['data']['furnishing']['furnishingName']
+            if furnishing_name not in involuntary_dissolution_stage_1_notification.PROCESSABLE_FURNISHING_NAMES:
+                raise QueueException('Furnishing name is not valid.')
```

Comment:
can we move

```python
            # Confirm the data.furnishingName
            furnishing_name = email_msg['data']['furnishing']['furnishingName']
            if furnishing_name not in involuntary_dissolution_stage_1_notification.PROCESSABLE_FURNISHING_NAMES:
                raise QueueException('Furnishing name is not valid.')
```

before 

`email = involuntary_dissolution_stage_1_notification.process(email_msg, token)`

Reply:
Moved